### PR TITLE
Add documentation for global-blocks

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Fixed tickets | none
| Related PRs | sulu/sulu#7229
| License | MIT

#### What's in this PR?

Add documentation for global-blocks

### CODE REVIEW

File: book/templates.rst [hunk 1]
```diff
@@ -885,6 +891,75 @@ You can also match multiple elements of different types. Use the wildcard
...
+
+Using Global blocks
+-------------------
+
+Another way to include other templates is to use global blocks. Global blocks define a set of properties that can be
+used as type inside of a block or any other property. This is useful if you want to reuse a block type in multiple
+templates.
+
+To define a global block, you have to create a new XML file in the ``config/templates/blocks`` directory. The following
+file is an example and defines a global block with the name ``text-block``:
```
Comment (Prokyonn) [inline book/templates.rst]:
```suggestion
file is an example and defines a global block with the key ``text_block``:
```

**Review by Prokyonn — APPROVED** [review]

File: book/templates.rst [hunk 2]
```diff
@@ -885,6 +891,75 @@ You can also match multiple elements of different types. Use the wildcard
         </properties>
     </template>
 
+.. _templates-global-blocks:
+
+Using Global blocks
+-------------------
+
+Another way to include other templates is to use global blocks. Global blocks define a set of properties that can be
```
Comment (chirimoya) [inline book/templates.rst]: `Another way to include other templates is to use global blocks.` remove.

Comment (wachterjohannes) [inline book/templates.rst]:
```suggestion
Global blocks define a set of properties that can be
```

File: book/templates.rst [hunk 3]
```diff
@@ -885,6 +891,75 @@ You can also match multiple elements of different types. Use the wildcard
...
+                <meta>
+                    <title lang="en">Description</title>
+                    <title lang="de">Beschreibung</title>
+                </meta>
+            </property>
+        </properties>
+    </template>
+
+This block can be used in any other template by using the ``<block>`` or ``<property>`` element with the ``type``
+attribute:
```
Comment (wachterjohannes) [inline book/templates.rst]:
```suggestion
node and the ``ref`` attribute:
```

Comment (chirimoya) [inline book/templates.rst]: within the ``type`` node and the ``ref`` attribute:

File: book/templates.rst [hunk 4]
```diff
@@ -885,6 +891,74 @@ You can also match multiple elements of different types. Use the wildcard
...
+.. _templates-global-blocks:
+
+Using Global blocks
+-------------------
+
+Global blocks define a set of properties that can be used as type inside of a block or any other property. This is
+useful if you want to reuse a block type in multiple templates.
+
+To define a global block, you have to create a new XML file in the ``config/templates/blocks`` directory. The following
+file is an example and defines a global block with the name ``text-block``:
```
Comment (Prokyonn) [inline book/templates.rst]:
```suggestion
file is an example and defines a global block with the name ``text_block``:
```

**Pull request merged**